BackofficeBundle/Resources/public/coffee/widget/AbstractConcurrancyCounter.coffee

```diff
@@ -0,0 +1,17 @@
+###*
+ * @class AbstractConcurrancyCounter
+###
+class AbstractConcurrancyCounter extends OrchestraView
+
+  addConcurrency: ->
+    if typeof @currentRankView == 'undefined'
+      @oldRender = @render
+      @currentRankView = 0
+    else
+      @currentRankView = @currentRankView + 1
+    @render = () ->
+      prototype = Object.getPrototypeOf(@)
+      if @currentRankView == prototype.currentRankView
+        @oldRender.apply this, arguments
```

Comment:
@oldRender is always defined as @currentRankView is a dynamically defined option and is always undefined as starting state.